addons/website/views/snippets/s_text_cover.xml

```diff
@@ -2,15 +2,17 @@
 <odoo>
 
 <template id="s_text_cover" name="Text Cover">
-    <section class="s_text_cover">
+    <section class="s_text_cover o_cc o_cc2">
         <div class="container-fluid">
-            <div class="row">
-                <div class="o_colored_level o_cc o_cc4 col-lg-5 pt160 pb160 px-5">
+            <div class="row o_grid_mode" data-row-count="11">
+                <div class="o_grid_item g-height-10 g-col-lg-5 col-lg-5 o_colored_level o_cc o_cc1" style="z-index: 1; grid-area: 2 / 3 / 11 / 8; --grid-item-padding-x: 24px; --grid-item-padding-y: 24px;">
                     <h1 class="display-3">Sell Online. <br/>Easily.</h1>
-                    <p class="lead">Write one or two paragraphs describing your product, services or a specific feature. To be successful your content needs to be useful to your readers.</p>
-                    <a t-att-href="cta_btn_href" class="btn btn-primary"><t t-esc="cta_btn_text">Contact us</t></a>
+                    <p class="lead"><br/>Sell online easily with a user-friendly platform that streamlines all the steps, including setup, inventory management, and payment processing.<br/></p>
+                    <a t-att-href="cta_btn_href" class="btn btn-lg btn-primary"><t t-esc="cta_btn_text">Contact us</t></a>
+                </div>
+                <div class="o_grid_item o_grid_item_image g-height-12 g-col-lg-6 col-lg-6 o_not_editable o_colored_level o_cc o_cc1 pt160 pb160 oe_img_bg d-none d-md-block" style="grid-area: 1 / 7 / 12 / -1; --grid-item-padding-x: 0px; --grid-item-padding-y: 0px;">
+                    <img class="img img-fluid mx-auto" src="/web/image/website.s_text_cover_default_image"/>
```

Comment:
Replacing the image is not available on this image - while it is available for the ones in `s_banner`. Also, drag and resize behave strangely.

Reply:
Not sure to understand why the image was moved within a `<img/>` tag at first place 🤔  I replicated the old behaviour instead, makes more sense IMO